src/components/MainContent.js

```diff
@@ -62,45 +70,188 @@ export default function MainContent() {
+      project: "Redesign New Online Shop",
+      members: [
+        {
+          profile: smiley,
+          name: "Jane Wane",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+        {
+          profile: smiley,
+          name: "Tiger Woods",
+          stackOrder: 1,
+          left: "left-[15px]",
+        },
+      ],
+      budget: "$2,000",
+      completion: 40,
+    },
+  ];
+
   return (
-    <div className="box w-screen h-screen grid grid-cols-2 overflow-y-auto bg-gray-200">
+    <div className="w-screen h-screen grid grid-cols-2 bg-gray-100 overflow-y-auto">
       <div className="w-[calc(100vw-238px)] h-full ml-[218px] grid grid-cols-6 auto-rows-min gap-5">
         <MainNavBar />
+
         {/* Company info section */}
         <div className="col-start-1 col-span-6 row-start-2 row-end-4 grid grid-cols-4 gap-4">
           {displayCompanyInfo}
         </div>
         {/* graphs section */}
-        <div className="col-start-1 col-span-4 row-start-4 row-end-7 border-[2px] border-black">
+        <div className="col-start-1 col-span-4 row-start-4 row-end-7">
           Graphs
         </div>
         {/* Project section */}
-        <div className="col-start-1 col-span-4 row-start-7 row-end-12 inline-block border-[2px] border-black">
-          <div className="h-[500px]">
-            <div className="pr-2 border-[1px] border-red-400 h-[85px]">
-              <h5 className="px-3 pt-4 font-semibold border-[1px] border-blue-400">
-                Projects
-              </h5>
-              <div className="flex justify-between px-3 pt-2 border-[1px] border-green-400">
-                <div className="flex items-center border-[1px] border-orange-600">
+        <div className="col-start-1 col-span-4 row-start-7 row-end-12 inline-block w-full h-fit rounded-lg bg-white">
+          <div className="h-[550px] px-1">
+            <div className="h-[85px] mb-3">
+              <h5 className="pt-4 px-3 text-lg font-bold">Projects</h5>
+              <div className="pt-2 px-3 flex justify-between">
+                <div className="flex items-center">
                   <ImCheckmark style={{ color: "blue" }} />{" "}
-                  <p className="text-xs">
-                    <span className="font-bold pl-1">30 done</span> this month
-                  </p>{" "}
+                  <p className="pl-2 text-sm text-gray-700">
+                    <span className="font-bold">30 done</span> this month
+                  </p>
                 </div>
-                <div className="border-[1px] border-yellow-400">
-                  <HiDotsVertical />
+                <div>
+                  <HiOutlineDotsVertical size="24" style={{ color: "gray" }} />
                 </div>
               </div>
             </div>
+            <div className="h-[calc(100%-97px)] w-[100%]">
+              <Table tableData={tableData} />
+            </div>
           </div>
         </div>
         {/* graph and orders section */}
```

Comment:
This would be a component of its own as well

Reply:
ok, will make it into one!